src/main/java/app/linguistai/bmvp/service/wordbank/UnknownWordService.java

```diff
@@ -454,4 +468,17 @@ private ConfidenceEnum decreaseConfidence(ConfidenceEnum currentConfidence) {
             ? ConfidenceEnum.values()[currentConfidence.ordinal() - 1]
             : currentConfidence;
     }
+
+    private void updateStatsBasedOnConfidence(ListStats stats, ConfidenceEnum confidence, Long count) {
```

Comment:
Instead of modifying the parameter, it is better to return the result and assign it in the used function

Reply:
Done